cli/internal/server/server_test.go

```diff
@@ -0,0 +1,72 @@
+package server
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-hclog"
+	"google.golang.org/grpc"
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/fs"
+
+	turbofs "github.com/vercel/turborepo/cli/internal/fs"
```

Comment:
Aside: we don't currently do this consistently. We should probably rename that package. Far better than clobbering the internal `fs`.

Reply:
I think that's a reasonable goal to have. There are also pieces in our `fs` that could stand to be their own package.

Reply:
Removing `fs.NewDir` let me get rid of these